tests/Behat/Foundry.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Behat support for Foundry
```

Comment:
Also probably add an example of using backgrounds to implement the fixtures. I use the back ground to set up core fixtures (that are used in all tests). 
https://behat.org/en/latest/user_guide/writing_scenarios.html#backgrounds